packages/ui/src/input/index.tsx

```diff
@@ -8,7 +8,7 @@ import useInputStyle from './styles';
 
 export * from './types';
 
-const Input = forwardRef<any, InputProps>((props, ref) => {
+export const Input = forwardRef<any, InputProps>((props, ref) => {
   const {
     as,
     'aria-label': ariaLabel,
```

Comment:
I would pull out `style` from `props` so that we can merge objects if we someone passes custom style props to the input:

```jsx
  const {
    as,
    'aria-label': ariaLabel,
    'aria-describedby': ariaDescribedby,
    isReadOnly,    
    style = {},
    ...rest
  } = props;
```

```jsx
  return (
    <PseudoBox
      ref={ref}
      as={as}
      ...
      style={{ '-webkit-appearance': 'none', ...style }}
```